content/_posts/falix/dashboard/2024-07-26-subusers.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: Learn how to manage your Subusers
+category: Dashboard
+tags: Server
+permalink: falix/dashboard/server/subusers
+description: Learn how to manage and create Subusers.
+author: TheTWIXhunter & Duolingo
+toc: true
+---
+
+## Information about subusers
+
+{: .info}
+> You do not need to add Sub-users to let other people start your server.
+> To let users (and even non-users) start your server, please let them use [the external start page](https://falixnodes.net/start)
+> or refer to our [External Start Guide](https://example.com).

```

Comment:
😃

Reply:
I fixed it (remind me to change it when the external start page gets moved